controllers/controller_utils.go

```diff
@@ -0,0 +1,17 @@
+package controllers
+
+import "time"
+
+type ResyncPeriodFunc func() time.Duration
+
+// NoResyncPeriodFunc returns 0 for resyncPeriod in case resyncing is not needed.
+func NoResyncPeriodFunc() time.Duration {
+	return 0
+}
+
+// StaticResyncPeriodFunc returns the resync period specified
+func StaticResyncPeriodFunc(resyncPeriod time.Duration) ResyncPeriodFunc {
```

Comment:
It looks like that this file is unused.

![image](https://user-images.githubusercontent.com/1450685/125224447-3abe7e80-e300-11eb-83f4-efafd210e9ec.png)

https://github.com/JohnNiang/ks-devops/search?q=StaticResyncPeriodFunc&type=code

Reply:
This file was created for controller test. But now, it's unnecessary. I will remove it immediately.